attributes/default.rb

```diff
@@ -15,3 +15,7 @@
 
 # PGP Key Server
 default['le']['pgp_key_server'] = 'pgp.mit.edu'
+
+# Debian Release
+
+default['le']['deb'] = 'trusty'
```

Comment:
K, i have changed the code please review...


*~Costya Regev**,DevOps,Totango*


*Website:*http://www.totango.com | *Mobile:* (+972)545722572 | *Skype: *
costya.regev

On Tue, Jun 9, 2015 at 11:53 AM, Joe Heung <notifications@github.com> wrote:

> In attributes/default.rb
> <https://github.com/logentries/le_chef/pull/38#discussion_r31993775>:
>
> > @@ -15,3 +15,7 @@
> >
> >  # PGP Key Server
> >  default['le']['pgp_key_server'] = 'pgp.mit.edu'
> > +
> > +# Debian Release
> > +
> > +default['le']['deb'] = 'trusty'
>
> Sorry for the delay in getting back to you, I'd set the attribute to
> default to the following:
>
> default['le']['deb'] =  node['lsb']['codename']
>
> That way users can override if they wish and this doesnt change the
> current behavior.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/logentries/le_chef/pull/38/files#r31993775>.
>